src/formats.ts

```diff
@@ -65,6 +65,14 @@ export function isSupportedLocale(locale: string): locale is SupportedLocale {
   return Object.values(SupportedLocale).indexOf(locale) !== -1;
 }
 
+export function getSafeLocaleFormat(locale) {
+  const [country] = locale.split('-');
```

Comment:
We need to make sure we check the original locale as well - for example, we support `pt-BR` specifically, and pulling just the country off makes us fail that check.